pages/index.js

```diff
@@ -1,25 +1,47 @@
+import { useEffect, useState } from "react";
 import styled from "styled-components";
-import FoodCard from "../components/FoodCard";
-import useSWR from "swr";
-import { fetcher } from "../helpers/api";
-
-export default function Home() {
-  const { data: recipes, error } = useSWR("/api/recipes", fetcher);
-
-  if (error) return <h1>There was an error</h1>;
-
-  if (!recipes) return <h1>your tasty recipes loading ...</h1>;
+import ChooseFoodCard from "../components/ChooseFoodCard";
 
+function Home({ nextRecipe, onAddRecipesToFavourites, onFindRandomRecipe }) {
   return (
     <main>
       <h1>WTF? What to food </h1>
-      <h2>my super tasty selection of recipes:</h2>
+      <h2>what are you up for?</h2>
+
+      <ChooseFoodCard
+        recipe={nextRecipe}
+        onAddRecipesToFavourites={onAddRecipesToFavourites}
+        onNextRecipe={onFindRandomRecipe}
+      />
 
-      <ul>
-        {recipes?.map((recipe) => (
-          <FoodCard recipe={recipe} key={recipe.id} />
-        ))}
-      </ul>
+      <SytledNav>
+        <StyledImgBox href="/">
+          <img src="/icons/iconHome.png" />
+        </StyledImgBox>
+        <StyledImgBox href="/mealcenter">
+          <img src="/icons/iconMealCenter.png" />
+        </StyledImgBox>
+      </SytledNav>
     </main>
   );
 }
+
+export default Home;
+
+const SytledNav = styled.nav`
+  border: 1px solid black;
+  border-radius: 10px;
+
+  display: flex;
+  justify-content: center;
+  width: 70%;
+  position: fixed;
+  left: 60px;
+  bottom: 5px;
+  background-color: #d9d1ce;
+`;
+
+const StyledImgBox = styled.a`
```

Comment:
Please think about a different name here, as this one does not really indicate that this is an `a`. Also, I recommend to use the `Link` component here.
```suggestion
const StyledLink = styled(Link)`
```